### PR TITLE
Combined Babel + DDL parser using the Calcite Babel parser

### PR DESCRIPTION
**Pull request by mihaibudiu**

### CODE REVIEW

File: sql-to-dbsp-compiler/SQL-compiler/src/main/codegen/includes/parserPostgresqlImpls.ftl [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+  SqlNode e;
+}
+{
+  { s = span(); }
+  e = PostgresqlSqlBeginTransactionMode() { s = span(); list = startList(e); }
+  ( <COMMA> e = PostgresqlSqlBeginTransactionMode() { list.add(e); } )*
+  { return new SqlNodeList(list, s.end(this)); }
+}
+
+SqlNode PostgresqlSqlBeginTransactionMode():
```
Comment (lalithsuresh) [inline sql-to-dbsp-compiler/SQL-compiler/src/main/codegen/includes/parserPostgresqlImpls.ftl:194]: Why do we need transaction modes in the grammar?

Comment (mihaibudiu) [reply sql-to-dbsp-compiler/SQL-compiler/src/main/codegen/includes/parserPostgresqlImpls.ftl:194]: I imported everything from Babel, will make it easier to merge future changes.

File: sql-to-dbsp-compiler/SQL-compiler/src/main/codegen/includes/parserImpls.ftl [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    <IF> <NOT> <EXISTS> { return true; }
+|
+    { return false; }
+}
+
+TableCollectionType TableCollectionTypeOpt() :
+{
+}
+{
+    <MULTISET> { return TableCollectionType.MULTISET; }
```
Comment (lalithsuresh) [inline sql-to-dbsp-compiler/SQL-compiler/src/main/codegen/includes/parserImpls.ftl:69]: Do we need these?

Comment (mihaibudiu) [reply sql-to-dbsp-compiler/SQL-compiler/src/main/codegen/includes/parserImpls.ftl:69]: Not currently, but these may be useful. But see the comment above.

**Review by lalithsuresh — APPROVED** [review]

**Pull request merged**